pkg/collector/corechecks/containers/containerd.go

```diff
@@ -177,6 +177,7 @@ func computeMetrics(sender aggregator.Sender, cu cutil.ContainerdItf, fil *ddCon
 		return
 	}
 
+	currentTime := time.Now()
```

Comment:
probably a nitpick but setting currentTime before the loop, while calling `cu.Info()` in the loop might make the timing inaccurate. And in the cri check you get the time closer to the container uptime computation: https://github.com/DataDog/datadog-agent/pull/4753/files#diff-42a33bd390107e0c8b42f393b5946a9cR117 - we might want to harmonize that.

Reply:
Yes you're right. I did it on purpose to be as accurate as possible, checking if calls trigger API calls or not, but in fact `cu.Info()` does trigger an API call, so we should get the time there.
Fixed.